Gemfile

```diff
@@ -79,13 +79,15 @@ group :development, :test do
   gem 'pry-byebug'
   gem 'rspec-rails', '~> 3.5.2'
   gem 'slim_lint'
+  gem 'teaspoon-mocha'
   gem 'thin'
 end
 
 group :test do
   gem 'axe-matchers'
   gem 'capybara-screenshot', github: 'mattheworiordan/capybara-screenshot'
   gem 'codeclimate-test-reporter', require: false
+  gem 'coffee-script'
```

Comment:
The interaction between sprockets 3 and teaspoon requires this gem to be loaded as well. I think teaspoon wrote its runner in coffee script, but decided not to compile it to JS before packaging it as a gem. Annoying. Seems like a wontfix on their end. Here is the issue https://github.com/jejacks0n/teaspoon/issues/405